provider/datagen/tests/data/json/units/displaynames@1/ar-EG-x-day-person.json

```diff
@@ -0,0 +1,5 @@
+{
+  "long": {},
+  "short": {},
+  "narrow": {}
+}
```

Comment:
@sffc and @robertbastian: those are empty because there is no locale data associated with them, there is only gender data.